src/web/lite.jsx

```diff
@@ -1,37 +1,48 @@
 import 'babel-polyfill'
 import React from 'expose-loader?React!react'
 import ReactDOM from 'expose-loader?ReactDOM!react-dom'
+import { connect } from 'react-redux'
+import { Provider } from 'react-redux'
 
+import store from './store'
+import { fetchModules } from './actions'
 import InjectedModuleView from '~/components/PluginInjectionSite/module'
+import { parseQueryString } from './util'
+import { moduleViewNames } from './util/Modules'
 
-const parseQueryString = () => {
-  const queryString = (window.location.search || '').substring(1) || ''
+const { m, v } = parseQueryString((window.location.search || '').substring(1) || '')
 
-  const params = {}
-  const queries = queryString.split('&')
-
-  for (let i = 0, l = queries.length; i < l; i++) {
-    const temp = queries[i].split('=')
-    params[temp[0]] = temp[1]
+class LiteView extends React.Component {
+  componentDidMount() {
+    this.props.fetchModules()
   }
-  return params
-}
 
-const { m, v } = parseQueryString()
+  render() {
+    const modules = moduleViewNames(this.props.modules.filter(module => module.loadInLiteMode))
+    const onNotFound = () => (
+      <h1>
+        Module ${m} with view ${v} not found
+      </h1>
+    )
 
-const LiteView = props => {
-  return (
-    <InjectedModuleView
-      moduleName={m}
-      viewName={v}
-      lite={true}
-      onNotFound={() => (
-        <h1>
-          Module ${m} with view ${v} not found
-        </h1>
-      )}
-    />
-  )
+    return (
+      <div>
+        <InjectedModuleView moduleName={m} viewName={v} lite={true} onNotFound={onNotFound} />
```

Comment:
Can you clarify this please? We render the specific module specified in the query string _and also_ all the modules we can find?

Reply:
Not all, only those that have `loadInLiteMode` flag.

Reply:
I see. But what is the use-case for the `m/v` params? Wouldn't the same module be loaded as part of the other modules?

Reply:
Module specified in the URI (like `/lite/?m=platform-webchat&v=fullscreen`) will be loaded in lite-mode as main UI-component. Modules with `loadInLiteMode` are intended to modify it when appropriate but can't serve as main components in lite-mode.

I think I'd rather rename `loadInLiteMode` to `isPlugin`. Would it make sense?

Reply:
Oh, now it makes total sense. My problem was there's also "lite" edition of botpress itself, right?

Reply:
Also why the main module is getting the same flag? If 'lite' is for plugins. Lots of confusion for me lol :)

Reply:
Sorry. I also got confused initially. :)
`isPlugin` (previously `loadInLiteMode`) is flag for plugins. `lite` is mode that some none-Plugin modules can run in.

Reply:
:)

Reply:
Haha, great points, I can see why you're confused. What about `lightweight` and `isLightweightPlugin`?

Reply:
@slvnperron , will we have non-lightweight plugins? ))
By the way what does `plugins` key in settings stand for? It could be another confusion. )

Reply:
@epaminond Not sure, I don't recall any `plugins` in the code base?

Reply:
@slvnperron , I was talking about this one: https://github.com/botpress/botpress-platform-webchat/blob/master/package.json#L12

Reply:
That name seems a bit confusing. )